flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/functions/TimeFunctionsITCase.java

```diff
@@ -804,4 +813,163 @@ private Stream<TestSetSpec> floorTestCases() {
                                         .format(TIMESTAMP_FORMATTER),
                                 STRING().nullable()));
     }
+
+    private Stream<TestSetSpec> toTimestampLtzTestCases() {
+        return Stream.of(
+                TestSetSpec.forFunction(BuiltInFunctionDefinitions.TO_TIMESTAMP_LTZ)
+                        .onFieldsWithData(
+                                100,
+                                1234,
+                                -100,
+                                null,
+                                DecimalDataUtils.castFrom(-Double.MAX_VALUE, 38, 18),
```

Comment:
I don't see here tests for invalid input
like invalid timezone, invalid format
invalid something else...

Reply:
Hi @snuyanzin , I have the invalid test cases here, the invalid inputs are directly pased in.  https://github.com/apache/flink/pull/25763/files/982b92f13fbb93b8e0c5244b30d4ae8bbbfa3a25#diff-7588925e682b7f5b4b2e41fdc4992ef0d9f246ce00696f783c36f993b8aeb7aeR950-R973

Could you clarify what you mean in this comment?

Reply:
Resolved.